src/Tasks/Microsoft.NET.Build.Tasks/targets/Microsoft.NET.Sdk.FrameworkReferenceResolution.targets

```diff
@@ -50,6 +51,21 @@ Copyright (c) .NET Foundation. All rights reserved.
     </CreateWindowsSdkKnownFrameworkReferences>
   </Target>
 
+  <PropertyGroup Condition="'$(EnablePrunePackageReferences)' == ''">
```

Comment:
This is intentionally a new property that can be disabled on the SDK side correct? 

Should we include Platform, or SDK in the name? Right now it's pretty generic.

Reply:
Yes, we can probably improve the name.  Maybe `PrunePlatformPackageReferences` would make sense.

@baronfel if you have any feedback.

Reply:
fwiw, the NuGet specific enabling is done by setting: RestoreEnablePackagePruning. 

I'm wondering if we should just key off that instead?

This would mean you can't enable pruning, but skip the references to prune from the SDK, but I can't imagine that's gonna be a common scenario.

Reply:
Yes, I don't think we need two different properties.  I've updated this PR to just key off of the NuGet one.